src/components/CopyCard/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { Fragment, useState } from "react";
+import { useHistory } from "react-router-dom";
+import { Card, Button, Snackbar, Tooltip } from "@material-ui/core";
+
+import { FileCopyOutlined } from "@material-ui/icons";
+
+export default function CopyCard(props) {
+  const { address } = props;
+  const [open, setOpen] = useState(false);
+  const history = useHistory();
+  const handleClick = () => {
+    setOpen(true);
+    navigator.clipboard.writeText(address);
+  };
+  const candidateRoute = () => {
```

Comment:
Let's make this `handleTextClick` and pass handler `history.push(`/candidate/${address}`);` as a prop from outside. So this same component can be used for other porposes as well if needed